README.md

```diff
@@ -1,6 +1,7 @@
 # scratch-resources
 Source files and images for translators to translate Scratch materials, published
 files for www.
+[![CircleCI](https://circleci.com/gh/LLK/scratch-resources/tree/develop.svg?style=shield&circle-token=8bf55740611abf98152feec3797bac5ee4149c7c)](https://circleci.com/gh/LLK/scratch-resources?branch=develop)
 
 ## For Scratch Team ##
 After updating any content to be published make sure to flush the fastly cache so that new versions are served, as the resources are cached aggressively.
```

Comment:
This comment is out of date now that we flush the cache as part of the deploy. Perhaps it would be better to describe what's happening now for the Scratch Team.

As part of the build process, a script generates the json for scratch-www listing all the resources, and any translated versions by locale. If the repo builds successfully, everything is uploaded to S3, and the fastly cache is flushed to ensure that new versions are served.